config/paths-common.conf

```diff
@@ -8,8 +8,10 @@ after  = paths-overrides.local
 [DEFAULT]
 
 sshd_log = %(syslog_authpriv)s
+sshd_backend = auto
```

Comment:
may be it is worth introducing `default_backend = auto` and then reusing that one as default for all so with a single change all could be changed, instead of per each one